### PR TITLE
clean up use of word: simply

### PR DESCRIPTION
**Pull request by kbhawkey**
- Create replacement text for word: simply.
- Related to #17070.

### CODE REVIEW

File: content/en/docs/concepts/cluster-administration/manage-deployment.md [hunk 1]
```diff
@@ -265,7 +265,7 @@ For a more concrete example, check the [tutorial of deploying Ghost](https://git
 ## Updating labels
 
 Sometimes existing pods and other resources need to be relabeled before creating new resources. This can be done with `kubectl label`.
-For example, if you want to label all your nginx pods as frontend tier, simply run:
+For example, if you want to label all your nginx pods as frontend tier, run:
```
Comment (celestehorgan) [inline content/en/docs/concepts/cluster-administration/manage-deployment.md:268]:
```suggestion
Hello world 
```

@kbhawkey (ignore this, i'm demonstrating for someone ;))

File: content/en/docs/tasks/configure-pod-container/translate-compose-kubernetes.md [hunk 2]
```diff
@@ -347,12 +341,14 @@ pod/redis-master-1432129712-63jn8   1/1           Running       0            4m
...
 
-**Note**:
+{{< note >}}
 
 - You must have a running Kubernetes cluster with a pre-configured kubectl context.
 - Only deployments and services are generated and deployed to Kubernetes. If you need different kind of resources, use the `kompose convert` and `kubectl apply -f` commands instead.
+{{< /note >}}
 
 ### OpenShift
+
```
Comment (kbhawkey) [inline content/en/docs/tasks/configure-pod-container/translate-compose-kubernetes.md:351]: Note: There is a bit more cleanup to do in this page.

**Review by irvifa — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**